idp/ui/src/app.css

```
@@ -112,3 +118,7 @@ strong {
 .oc-light {
   color: #fff !important;
 }
+
+.oc-login-form div:first-of-type {
```

```suggestion
.oc-login-form div:not(:last-of-type) {
```
If we would add more inputs in the future (haha, for a login page probably not gonna happen) the margin-bottom is needed for all except the last.